src/tests/unit/tests/injected/components/__snapshots__/issue-details-navigation-controls.test.tsx.snap

```diff
@@ -1,91 +1,203 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`IssueDetailsNavigationControls render handles button states: {"backButtonDisabled": false, "nextButtonDisabled": false} 1`] = `
-<div
-  className="insights-dialog-next-and-back-container"
->
-  <div>
-    <CustomizedDefaultButton
-      data-automation-id="back"
-      onClick={[Function]}
-      text="< Back"
-    />
+<DocumentFragment>
+  <div
+    class="insights-dialog-next-and-back-container"
+  >
+    <div>
+      <button
```

Comment:
guess we can mock CustomizedDefaultButton

Reply:
Implemented the suggested changes.